src/llms/prompts.py

```diff
@@ -76,6 +76,7 @@
    - Do not lose any commit information, even if it doesn't match any category
 
 3. **Entry Format:**
+   - Ignore the pull request that has title as "[AUTO] Increment version to".
```

Comment:
Add a filter messages section and then put this, will be easy to maintain all future filtering requirements.

Reply:
Makes sense. Made that change